Sources/Identity/CustomerInfo.swift

```diff
@@ -129,13 +119,12 @@ import Foundation
             return false
         }
 
-        return NSDictionary(dictionary: self.jsonObjectWithNoDate)
```

Comment:
Much simpler comparison! `CustomerInfoResponse` `Equatable` implementation ignores the date.